### ISSUE TITLE
Filter not working as it should and potential to cause a a lot of work in terms of clean up work

### ISSUE THREAD

**Issue by oskarsword** [DataDog/datadog-sync-cli#281]
**Overview of issue:**

**Steps to reproduce the issue:**
1. Import with a filter in a config file
2. Sync with a filter in a config file

**Describe the results you received:**
For some reason it imports everything and it also sync everything over.
Now I have a massive clean up job to do because of it.

**Describe the results you expected:**
I expected the filter to work.
Meaning it only syncs over the stuff I actually put in the filter.
What else is the use for the filter?

**Additional information you deem important (e.g. issue happens only occasionally):**

**Comment by oskarsword** [comment]
I got a bit upset when I wrote this issue after importing over 300 resources I did not want into the destination organization.
But to me it would make sense that if I have a filter applied then no other resource types would be imported but that was not the case. It seems as if the filter only applies to the resource specified and simply imports all resources of all other types. 
This means I would have to manually remove all other resource types json or somehow make a filter for everyone single resource type with some obscure value that would never exist.

I think there should be a way to specify somehow if a filter should exclude all other resource types or something along the lines of allowing wildcards for resource types. Maybe there is something I am missing.
